templates/zerver/integrations/google-calendar.md

```diff
@@ -1,72 +1,67 @@
+# Zulip Google Calendar integration
+
 Get Google Calendar reminders in Zulip! This is a great way to see
 your reminders directly in your Zulip feed.
 
-1.  {!download-python-bindings.md!}
+!!! tip ""
 
     This bot should be set up on a trusted machine, because your API
     key is visible to local users through the command line or config
     file.
 
-1.  Next, follow the instructions for **Step 1** at
-    [this link](https://developers.google.com/google-apps/calendar/quickstart/python)
-    to get a `client_secret` file. Save this file as `client_secret.json`
-    to your `~/` directory.
+{start_tabs}
+
+1.  {!download-python-bindings.md!}
 
-1.  Next, install the latest Google API Client for Python by following the
-    instructions on the
-    [Google website](https://developers.google.com/api-client-library/python/start/installation).
+1.  Follow [Google's instructions][google-quickstart] to get a
+    `client_secret` file, and save it as `client_secret.json` in your
+    `~/` directory.
 
-1.  In Zulip, go to your click on the cog in the top right corner, and
-    then clicking on **Personal settings**.
+1.  Install the latest Google API Client for Python by following [Google's
+    instructions][google-api-client].
 
-1.  Click on the tab that’s labeled **Account & privacy** and click on
-    **Manage your API key**. Enter your password if prompted, and
-    download the `zuliprc` file. Save this file as `.zuliprc` to your `~/`
+1.  In Zulip, click the **gear** (<i class="fa fa-cog"></i>) icon at the top
+    right, select **Personal settings**. Select the **Account & privacy**
+    tab, and click **Manage your API key**. Enter your password if prompted,
+    download the `zuliprc` file, and save it as `.zuliprc` in your`~/`
     directory.
 
-    ![Download zuliprc file](/static/images/integrations/google/calendar/001.png)
+1.  To authorize your Zulip bot to view your Calendar, run the script:
+
+    `python /usr/local/share/zulip/integrations/google/get-google-credentials`
+
+    Authorize access in the browser window that opens. If no browser opens,
+    follow the terminal prompts.
 
-1.  Run the `get-google-credentials` with this command:
+1.  In the same directory, start the Zulip bot with the command:
 
-        python /usr/local/share/zulip/integrations/google/get-google-credentials
+    `python /usr/local/share/zulip/integrations/google/google-calendar --user <email for reminders>`
 
-1.  It should open up a browser and ask you for certain permissions. Give
-    Zulip access, and move on to the next step. If it doesn’t open a
-    browser, follow the instructions in the terminal window.
+    Use the [optional flags](#configuration-options) to configure the bot.
 
-1.  Now, all that’s left to do is to run the `gcal-bot` script, in the
-    same directory as the `get-google-credentials` script, with the
-    necessary parameters:
+1.  You will only get reminders as long as the terminal session with the bot
```

Comment:
Re-added this paraphrased point, separating it from the tip.

Reply:
I think combining these two points is better:

> You will get reminders as long as the terminal session with the bot  remains open. Consider using `screen` to run the bot in the background.